src/test/java/com/wikia/webdriver/common/remote/operations/PostRemoteOperation.java

```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.common.remote.operations;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import lombok.AllArgsConstructor;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.AllArgsConstructor'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)